internal/provider/resource_schema_version.go

```diff
@@ -0,0 +1,234 @@
+package provider
+
+import (
+	webclient "axual-webclient"
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/dcarbone/terraform-plugin-framework-utils/validation"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+var(
+	_ tfsdk.ResourceType = schemaVersionResourceType{}
+	_ tfsdk.Resource = schemaVersionResource{}
+ 	_ tfsdk.ResourceWithImportState = schemaVersionResource{}
+)
+
+type schemaVersionResourceType struct{}
+
+func (r schemaVersionResourceType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+		MarkdownDescription: "Schema version resource. Read more: https://docs.axual.io/axual/2023.2/self-service/schema-management.html",
+
+		Attributes: map[string]tfsdk.Attribute{
+			"body": {
+				MarkdownDescription: "Avro schema",
+				Required:            true,
+				Type:                types.StringType,
+				
+			},
+			"version": {
+				MarkdownDescription: "The version of the schema",
+				Required:            true,
+				Type:                types.StringType,
+			},
+			"description": {
+				MarkdownDescription: "A short text describing the Schema version",
+				Optional:            true,
+				Type:                types.StringType,
+				Validators: []tfsdk.AttributeValidator{
+					validation.Length(0, 500),
+				},
+			},
+			"schema_version_id": {
```

Comment:
Can we keep this as id. I would prefer it this way as it's consistent with all our other resources.